paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -189,6 +198,29 @@ class CustomerSheet @Inject internal constructor(
                 .billingDetailsCollectionConfiguration(billingDetailsCollectionConfiguration)
         }
 
+        override fun hashCode(): Int {
```

Comment:
Should we bring in poko instead?

Reply:
Sure! I’ll follow up with a pull request to do that across our SDKs.